samples/crypto/aes_cbc/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+# Override the binary used by qemu
+set(QEMU_KERNEL_OPTION "-device;loader,file=${CMAKE_BINARY_DIR}/tfm_qemu.hex")
```

Comment:
Question: isn't it sufficient to set `QEMU_KERNEL_FILE` ?

Secondly, what defines that the name of the hex will be `tfm_qemu.hex` ?

Also add tome more comments regarding:
- Why does this have to be replaced
- Where is / who defines the hex feile name to be `tfm_qemu.hex`

Reply:
@Vge0rge?

Reply:
This is actually a leftover from the previous configuration of the tfm_hello world I think. I will remove that!

Reply:
Fixed